src/Microsoft.TemplateEngine.Orchestrator.RunnableProjects/SimpleConfigModel.cs

```diff
@@ -559,59 +606,57 @@ private IGlobalRunConfig ProduceOperationSetup(SpecialOperationConfigParams defa
                         }
                     }
 
-                    if (symbol.Value.Replaces != null)
+                    string sourceVariable = null;
+                    if (string.Equals(symbol.Value.Type, SymbolModelConverter.BindSymbolTypeName, StringComparison.Ordinal))
                     {
-                        string sourceVariable = null;
-
-                        if (string.Equals(symbol.Value.Type, SymbolModelConverter.BindSymbolTypeName, StringComparison.Ordinal))
+                        if (string.IsNullOrWhiteSpace(symbol.Value.Binding))
                         {
-                            if (string.IsNullOrWhiteSpace(symbol.Value.Binding))
-                            {
-                                EnvironmentSettings.Host.LogMessage($"Binding wasn't specified for bind-type symbol {symbol.Key}");
-                            }
-                            else
-                            {
-                                //Since this is a bind symbol, don't replace the literal with this symbol's value, 
-                                //  replace it with the value of the bound symbol
-                                sourceVariable = symbol.Value.Binding;
-                            }
+                            EnvironmentSettings.Host.LogMessage($"Binding wasn't specified for bind-type symbol {symbol.Key}");
                         }
                         else
                         {
-                            //Replace the literal value in the "replaces" property with the evaluated value of the symbol
-                            sourceVariable = symbol.Key;
+                            //Since this is a bind symbol, don't replace the literal with this symbol's value, 
+                            //  replace it with the value of the bound symbol
+                            sourceVariable = symbol.Value.Binding;
                         }
+                    }
+                    else
+                    {
+                        //Replace the literal value in the "replaces" property with the evaluated value of the symbol
+                        sourceVariable = symbol.Key;
+                    }
 
-                        if (sourceVariable != null)
+                    if (sourceVariable != null)
+                    {
+                        if (symbol.Value is BaseValueSymbol p)
```

Comment:
I noticed that `sourceVariable` isn't used in this `if` block. Should this be outside of the `if (sourceVariable != null)` block or is this correct?

I'm OK if you resolve this comment without a reply. Just checking to make sure.